src/helpers/iterators.ts

```diff
@@ -76,3 +78,6 @@ export async function* asyncIterableToGenerator<T>(iter: AsyncIterable<T>) {
     yield entry;
   }
 }
+
+/** Promisify a pipeline */
+export const pipelineAsync = util.promisify(stream.pipeline);
```

Comment:
This is available now:
```suggestion
export { pipeline as pipelineAsync } from 'node:stream/promises';
```

Reply:
Oh cool, done